docs/content/users/quickstart.md

```diff
@@ -192,6 +192,23 @@ ddev launch
 
     When the installation wizard prompts for database settings, enter `db` for the _DB Server Address_, _DB Name_, _DB Username_, and _DB Password_.
 
+## Ibexa DXP
+
+Install [Ibexa DXP](https://www.ibexa.co) OSS Edition.
+
+```bash
+mkdir my-ibexa-project && cd my-ibexa-project
+ddev config --project-type=php --php-version 8.1 --docroot=public --create-docroot
+ddev config --web-environment-add DATABASE_URL=mysql://db:db@db:3306/db
```

Comment:
It seems like another option would be to add this to the project-level .env? It can also be added to .ddev/.env, which may be a little easier than this.

Of course this works fine.